src/encoded/types/file.py

```diff
@@ -476,6 +479,33 @@ def s3_uri(self):
     def library(self, request, replicate):
         return request.embed(replicate, '@@object?skip_calculated=true').get('library')
 
+    @calculated_property(schema={
+        "title": "Assay title",
+        "type": "string",
+        "notSubmittable": True,
+    })
+    def assay_title(self, request, dataset):
+        dataset = request.embed(dataset, '@@object')
+        return dataset.get('assay_title')
+
+    @calculated_property(schema={
+        "title": "Biosample ontology",
+        "type": "string",
+        "notSubmittable": True,
+    })
+    def biosample_ontology(self, request, dataset):
+        dataset = request.embed(dataset, '@@object')
+        return dataset.get('biosample_ontology')
+
+    @calculated_property(schema={
+        "title": "Target",
+        "type": "string",
+        "notSubmittable": True,
+    })
+    def target(self, request, dataset):
+        dataset = request.embed(dataset, '@@object')
+        return dataset.get('target')
```

Comment:
I think this should just be target.label?

Reply:
K, thought maybe there was some other interesting stuff in target but should probably try to keep it as slim as possible.